### PR TITLE
feat: editTrigger prop for editable Paragraph

### PR DESCRIPTION
**Pull request by kachanovskiy**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [X] Site / documentation update
- [X] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [X] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution
in order to be able to use Paragraph as true inline edit (with no extra elements), it may be useful to hide the icon and use the value itself as edit trigger.

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | for editable Paragraph, edit can be triggered by clicking the icon or the value itself (or both) |
| 🇨🇳 Chinese | 对于可编辑段落，可以通过点击图标或值本身（或两者都有）来触发进入编辑状态。 |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [X] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

### CODE REVIEW

File: components/typography/index.zh-CN.md [hunk 1]
```diff
@@ -96,6 +96,8 @@ cover: https://gw.alipayobjects.com/zos/alicdn/GOM1KQ24O/Typography.svg
       onChange: function(string),
       onCancel: function,
       onEnd: function,
+      editTrigger: 'icon' | 'value' | 'both',
```
Comment (afc163) [inline components/typography/index.zh-CN.md:99]:
```suggestion
      triggerType: 'icon' | 'text' | 'both',
```

File: components/typography/index.zh-CN.md [hunk 2]
```diff
@@ -111,6 +113,8 @@ cover: https://gw.alipayobjects.com/zos/alicdn/GOM1KQ24O/Typography.svg
 | onStart | 进入编辑中状态时触发 | function | - |  |
 | onCancel | 按 ESC 退出编辑状态时触发 | function | - |  |
 | onEnd | 按 ENTER 结束编辑状态时触发 | function | - |  |
+| editTrigger | 编辑模式触发器 - 图标、值（文本）或两者皆有（在有“值”的情况下图标会被隐藏） | `icon`\|`value`\|`both` | `icon` |  |
+| alwaysShowEditIcon | 当 editTrigger = "value" 时被使用 - 图标默认是隐藏的，但可以将它显示出来 | boolean | false |  |
```
Comment (afc163) [inline components/typography/index.zh-CN.md:117]: 是否需要这个属性？我建议先不加

File: components/typography/index.zh-CN.md [hunk 3]
```diff
@@ -96,6 +96,7 @@ cover: https://gw.alipayobjects.com/zos/alicdn/GOM1KQ24O/Typography.svg
       onChange: function(string),
       onCancel: function,
       onEnd: function,
+      triggerType: 'icon' | 'text' | 'both',
```
Comment (afc163) [inline components/typography/index.zh-CN.md:99]:
```suggestion
      triggerType: ('icon' | 'text')[],
```

Comment (afc163) [reply components/typography/index.zh-CN.md:99]: `both` => `['icon', 'text']`

**Comment by afc163** [comment]
Conflicting

**Comment by afc163** [comment]
ping @kachanovskiy

**Comment by afc163** [comment]
CI is broken, could you check it?

**Comment by kachanovskiy** [comment]
I am sorry, I am unable to identify what's wrong - locally it builds just fine, and CI run log does not have enough of data I could have used to look into this. Could you please assist?

one thing i used to have at some point (and which then has gone) was errors due to postcss 8 required but not present...

**Comment by afc163** [comment]
```
npm test -- components/typography -u
git commit -am 'chore: update snapshot'
git push origin skachano_paragraph_editable_edit_trigger
```

**Comment by afc163** [comment]
图片

Test coverage is down.

**Comment by afc163** [comment]
ping @kachanovskiy

**Comment by kachanovskiy** [comment]
apologies, this would have to wait until i am back next thursday, i am away right now…sorry about this.
________________________________

**Comment by kachanovskiy** [comment]
@afc163 seems like i did something wrong, could you please help me with this? thanks...

**Comment by kachanovskiy** [comment]
ok seems like i did "fetch upstream" in a wrong branch, which has caused a merge of master into my branch in my fork.

am i getting it right that i can either:
1) do "git push --force" from my local copy where 8a46d6962005d26e5ffbfa84a8c1cfb369cd1dbd is still on top
2) or clone my repo from github, checkout my branch, do "git reset --hard HEAD~1" and then "git push --force" which is supposed to bring my repo back to where it was before my erroneous action?

**Comment by kachanovskiy** [comment]
@afc163 i have removed merge commit from the erroneous upstream fetch from the branch in my fork, please advise what should i do now - would you re-open this PR, or should i create a new one from the same branch and go through all the steps again?

**Comment by afc163** [comment]
You can create a new PR from scratch.

**Comment by xrkffgg** [comment]
Next can use https://github.com/ant-design/ant-design/issues/30441

**Pull request closed without merging**